tests/magick-image/gaussian-blur.spec.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright Dirk Lemstra https://github.com/dlemstra/magick-wasm.
+// Licensed under the Apache License, Version 2.0.
+
+import { Channels } from '@src/enums/channels';
+import { ErrorMetric } from '@src/enums/error-metric';
+import { TestImages } from '@test/test-images';
+
+describe('MagickImage#gaussianBlur', () => {
+    it('should gaussian blur the image', () => {
+        TestImages.Builtin.wizard.use(imageA => {
```

Comment:
Could you use `image, other` instead of `imageA, imageB`? This is used in the rest of the code.